pipx/animate.py

```diff
@@ -59,12 +60,19 @@ def print_animation(
     period: float,
     animate_at_beginning_of_line: bool,
 ):
+    (term_cols, _) = shutil.get_terminal_size(fallback=(9999, 24))
     while not event.wait(0):
         for s in symbols:
             if animate_at_beginning_of_line:
-                cur_line = f"{s} {message}"
+                if len(message) <= term_cols - 3:
```

Comment:
the numbers fill a bit cryptic: 3, 4, 6; can we pull them out into constants so that the code reads better 🤔

Reply:
I was trying to figure out a way to do this that wasn't overly obfuscated in making it symbolic.  This subtracted number to do with the length of the added ellipsis `...` the length of the other parts of the line `{s} ` and a `+1` because for some reason the last character is removed after the `\r` is printed.

Originally I just truncated the entire `cur_line` which was more self-explanatory, but that approach didn't work to cover the case of animated characters at the end. (because it would truncate the end animation).

In general you're right but I couldn't think of a clean way to write it.  I'll think some more.

Reply:
Changed it, I think the new code is a bit better